std_msgs/Header header
string[] labels
std_msgs/Float32MultiArray boxes
